packages/components/src/products.tsx

```diff
@@ -281,7 +279,7 @@ export const PRODUCTS_MENU_LIST: Record<string, { type: 'separator'; title: Reac
                   i > 6 && 'rounded-sm bg-gray-500 text-white dark:bg-white/10',
                 )}
               >
-                <Logo className="size-4 text-[8px]" />
+                <item.logo className="size-4 text-[8px]" />
```

Comment:
I remember an obscure bug where lowercase method component broke Fast Refresh. I'll try to find it.